data/migrations/versions/3383aad1e992_add_uploadedblob_table.py

```diff
@@ -23,7 +23,11 @@ def upgrade(op, tables, tester):
     if "uploadedblob" not in table_names:
         op.create_table(
             "uploadedblob",
-            sa.Column("id", sa.BigInteger(), nullable=False),
+            sa.Column(
+                "id",
+                sa.Integer() if op.get_context().dialect.name == "sqlite" else sa.BigInteger(),
```

Comment:
```
sa.Column(
    "id",
    sa.BigInteger().with_variant(sa.Integer, "sqlite"),
    nullable=False,
)
...
```
might be a bit cleaner option.

It is described at https://docs.sqlalchemy.org/en/20/dialects/sqlite.html#allowing-autoincrement-behavior-sqlalchemy-types-other-than-integer-integer